BasicsOOP/BankAccount.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BasicsOOP;
+/// <summary>
+/// Типы банковских счетов
+/// </summary>
+public enum BankAccountType
+{
+    Payment,
+    Current,
+    Deposit,
+    Currency,
+    Special,
+    Budget,
+}
+/// <summary>
+/// Класс банковского счёта
+/// </summary>
+public class BankAccount
+{
+    static private int TotalBankNumber = 0;
```

Comment:
```suggestion
    private static int TotalBankNumber = 0;
```
Обычно в такой последовательности должно быть написано.